### DISCUSSION TITLE
Fichiers non suivis

### DISCUSSION THREAD

**Discussion by etienne-64** [dbarzin/mercator#96] [Q&A] [answered]
Il y a beaucoup de fichiers non suivis. Est-ce normal ? Ne peuvent-ils / doivent-ils pas être ajoutés à .gitignore ?
![image](https://user-images.githubusercontent.com/10651729/165796933-83b673b9-fb17-43df-bf03-c7b1f3a853c7.png)

**Comment by dbarzin** [comment, marked as answer]
Ce sont des fichiers générés par le Laravel.
Il faudrait que je les ignore explicitement dans la configuration de Git.

**Comment by dbarzin** [comment]
Le fichier .gitignore a été complété.
